### ISSUE TITLE
Batch normalization running statistics

### ISSUE THREAD

**Issue by gaoyi-byte** [fmu2/PyTorch-MAML#11]
when i set the 

encoder_args:
  bn_args:
    track_running_stats: True

the train will not converge?
